Feature/nri html

MarieMatsumoto opened the pull request:
CSSファイル作成

MarieMatsumoto committed:
Create style.css

CSSファイル作成

MarieMatsumoto committed:
Create reset.css

リセットCSS作成

ykyk1218 commented:
<img width="1251" alt="NRIの強み：ビジネスモデルの特徴___企業情報___野村総合研究所_NRI_" src="https://user-images.githubusercontent.com/830861/94268354-013aa380-ff78-11ea-8db2-6dbc582120fb.png">

ykyk1218 commented:
<img width="440" alt="NRIの強み：ビジネスモデルの特徴___企業情報___野村総合研究所_NRI_" src="https://user-images.githubusercontent.com/830861/94268515-3f37c780-ff78-11ea-83c1-c5b9f1fa086e.png">

ykyk1218 commented:
`>` は テキストと色が違うのでお手本サイトを確認してみてください。

<img width="868" alt="NRIの強み：ビジネスモデルの特徴___企業情報___野村総合研究所_NRI_" src="https://user-images.githubusercontent.com/830861/94268925-df8dec00-ff78-11ea-960f-b209b7982e68.png">

ykyk1218 commented:
ここの < まで下線が入ってしまう問題は、a タグで < を表現しようとしているためなので、aタグ以外でやれば問題ないですね。

<img width="1014" alt="NRIの強み：ビジネスモデルの特徴___企業情報___野村総合研究所_NRI_" src="https://user-images.githubusercontent.com/830861/94269073-1fed6a00-ff79-11ea-9243-8f007565dcc3.png">

MarieMatsumoto commented:
> ここの<まで下線が入ってしまう問題は、aタグで<を表現しようとしているためなので、aタグ以外でやれば問題ないですね。
> 
> <img alt="NRIの強み：ビジネスモデルの特徴___企業情報___野村総合研究所_NRI_" width="1014" src="https://user-images.githubusercontent.com/830861/94269073-1fed6a00-ff79-11ea-9243-8f007565dcc3.png">

承知いたしました。liの疑似要素に変更します。ただそうすると「＞」部分ではリンクが反応しないのですが、それは正しいですか？（本家は「＞」にホバーするとリンクが反応します。）

MarieMatsumoto commented:
![](https://user-images.githubusercontent.com/830861/94268925-df8dec00-ff78-11ea-960f-b209b7982e68.png)

幅揃わない問題については再度格闘します。
＞を左に寄せる方法はまったくお手上げ状態です。記号を背景に表示させる方法があるのでしょうか？それともこの＞は画像ですか？（画像であれば解決できそうなのですが…）

MarieMatsumoto commented:
https://github.com/MarieMatsumoto/html-css-nri/pull/2#issuecomment-698907016

こちらについても、aに設定するとこんな感じになってしまいます。
<img width="316" alt="スクリーンショット 2020-09-25 22 29 57" src="https://user-images.githubusercontent.com/68460279/94273215-0f3ff280-ff7f-11ea-9e13-ec0f2dfc5ac8.png">
かといかといってpに設定すると枠外にはみ出てしまうのですが、記号を背景に表示させる方法があるのでしょうか…
（小林さんに頂いたコメントを直接編集してしまいました。申し訳ございません。今は元に戻っております）

MarieMatsumoto commented:
https://github.com/MarieMatsumoto/html-css-nri/pull/2#issuecomment-698906274

SVGの大きさは変更いたします。
文字位置の調整についても正直見当はついておりませんが格闘してみます。

ykyk1218 commented:
> > ここの<まで下線が入ってしまう問題は、aタグで<を表現しようとしているためなので、aタグ以外でやれば問題ないですね。
> > <img alt="NRIの強み：ビジネスモデルの特徴___企業情報___野村総合研究所_NRI_" width="1014" src="https://user-images.githubusercontent.com/830861/94269073-1fed6a00-ff79-11ea-9243-8f007565dcc3.png">
> 
> 承知いたしました。liの疑似要素に変更します。ただそうすると「＞」部分ではリンクが反応しないのですが、それは正しいですか？（本家は「＞」にホバーするとリンクが反応します。）

あー、なるほどです...
すいません、「＞」にホバーするとリンクが反応するところに気付かず...

となると対応方法としては a タグでやる必要が出てきますのね。
underline が出ないようにするには 「>」に hover した時、underline を打ち消す CSS を適用させるか、「>」を普通のテキスト以外で表現するかでしょうか。

ykyk1218 commented:
> ![](https://user-images.githubusercontent.com/830861/94268925-df8dec00-ff78-11ea-960f-b209b7982e68.png)
> 
> 幅揃わない問題については再度格闘します。
> ＞を左に寄せる方法はまったくお手上げ状態です。記号を背景に表示させる方法があるのでしょうか？それともこの＞は画像ですか？（画像であれば解決できそうなのですが…）

左に寄せる方法は...うまいヒント思いつかないので、そのまま伝えますが、`position: absolute;` を使うのが良さそうですね。

ykyk1218 commented:
> [#2 (comment)](https://github.com/MarieMatsumoto/html-css-nri/pull/2#issuecomment-698907016)
> 
> こちらについても、aに設定するとこんな感じになってしまいます。
> <img alt="スクリーンショット 2020-09-25 22 29 57" width="316" src="https://user-images.githubusercontent.com/68460279/94273215-0f3ff280-ff7f-11ea-9e13-ec0f2dfc5ac8.png">
> かといかといってpに設定すると枠外にはみ出てしまうのですが、記号を背景に表示させる方法があるのでしょうか…
> （小林さんに頂いたコメントを直接編集してしまいました。申し訳ございません。今は元に戻っております）

これも `position: absolute;` で解決しそうです 🙂

ykyk1218 commented:
ここのナビゲーションメニューの位置についてですが、状況的には
ul タグに flexbox をつけて、`align-items:center` を指定しているので li タグ自体は ul の天地中央に配置されています。
ただ li タグの中のテキストが中央にきていないので、li タグの中でテキストが天地中央に来るように指定する必要がありますね 😌

![image](https://user-images.githubusercontent.com/830861/94278282-96906480-ff85-11ea-836d-5de473255e75.png)

MarieMatsumoto commented:
> > > ここの<まで下線が入ってしまう問題は、aタグで<を表現しようとしているためなので、aタグ以外でやれば問題ないですね。
> > > <img alt="NRIの強み：ビジネスモデルの特徴___企業情報___野村総合研究所_NRI_" width="1014" src="https://user-images.githubusercontent.com/830861/94269073-1fed6a00-ff79-11ea-9243-8f007565dcc3.png">
> > 
> > 
> > 承そういたしました。liの疑似要素に変更します。ただそうすると「＞」部分ではリンクが反応しないのですが、それは正しいですか？（本家は「＞」にホバーするとリンクが反応します。）
> 
> あー、なるほどで​​す...
> すいません、「＞」にホバーするとリンクが反応するところに気付かず...
> 
> となると対応方法としてはタグでやる必要が出てきますのね。
> 下線が出ないようにするには「>」にホバーした時、下線を打ち消すCSSを適用させるか、「>」を普通のテキスト以外で表現するかでしょうか。

いえこちらこそしつこくすみません…。
＞にホバーした時に下線が出ないように
`.fb-mini a:hover::before {
    content: ">";
    padding-right: 10px;
    font-size: 10px;
    text-decoration-line: none !important;
}`
としてみましたがだめでした…。
最初に設定した
`a:hover {
    text-decoration: underline;
}`
を無効にするとうまくいくのですが、これを採用するとすべてのa要素についてホバーしたときに`text-decoration: underline;`を設定しないといけないですよね？

ちなみに
＞「>」を普通のテキスト以外で表現する
というのは文字コードということでしょうか…？（的外れでしたらすみません）

MarieMatsumoto commented:
> ここの
> ナビゲーションメニューの位置についてですが、状況的にはulタグにflexboxをつけて、`align-items:center`を指定しているのでliタグ自体はulの天地中央に配置されています。ただ
> liタグの中のテキストが中央にきていないので、liタグの中でテキストが天地中央に来るように指定する必要がありますね😌
> 
> ![画像](https://user-images.githubusercontent.com/830861/94278282-96906480-ff85-11ea-836d-5de473255e75.png)

ありがとうございます。a要素の上下にパディングを設定するとなんとなくそれっぽくなりました！（全然違ったらすみません）

MarieMatsumoto commented:
> > ![](https://user-images.githubusercontent.com/830861/94268925-df8dec00-ff78-11ea-960f-b209b7982e68.png)
> > 幅揃わない問題については再度格闘します。
> > ＞を左に寄せる方法はまったくお手上げ状態です。記号を背景に表示させる方法があるのでしょうか？それともこの＞は画像ですか？（画像であれば解決できそうなのですが…）
> 
> 左に寄せる方法は...うまいヒント思いつかないので、そのまま伝えますが、`position: absolute;`を使うのが良さそうですね。

ありがとうございます。承知いたしました。「デジタルが拓く〜」の部分についても同様に修正いたします。

MarieMatsumoto committed:
下部FollowUSに<strong>追加、<ul class="footer-link">追加

MarieMatsumoto committed:
Update style.css

ヘッダー固定部分にz-index: 100;追加、ヘッダー左上ロゴSVGの大きさ変更、グローバルナビ部分に上下パディングを設定し縦にセンタリング、キービジュアルに重なるテキストの位置を調整、メイン記事以下a要素の疑似要素beforeの背景にSVGファイルを表示させるよう設定

MarieMatsumoto committed:
Merge branch 'master' into feature/nri-html

MarieMatsumoto committed:
Rename style.css to CSS/style.css

MarieMatsumoto committed:
フォルダ名変更

MarieMatsumoto committed:
重複のため削除

ykyk1218 commented:
<img width="901" alt="NRIの強み：ビジネスモデルの特徴___企業情報___野村総合研究所_NRI_" src="https://user-images.githubusercontent.com/830861/94365014-7fc64b00-0108-11eb-8c33-ad22adea2c2e.png">

MarieMatsumoto committed:
Update style.css

.strengthの高さ変更、高さが一番高い要素に揃うようflexboxの子要素にdisplay: flex;、align-items: center;を追加、疑似要素が中心あたりに来るよう高さを変更

MarieMatsumoto committed:
Merge branch 'feature/nri-html' of https://github.com/MarieMatsumoto/html-css-nri into feature/nri-html

ykyk1218 commented:
@MarieMatsumoto 
バッチリです！
LGTM 🙆‍♂️
